bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -3747,57 +3745,70 @@ void ReplicaImp::executeRequestsInPrePrepareMsg(concordUtils::SpanWrapper &paren
       LOG_DEBUG(CNSUS, "Consensus reached");
     }
     SCOPED_MDC("pp_msg_cid", ppMsg->getCid());
+    std::deque<IRequestsHandler::ExecutionRequest> accumulatedRequests;
     while (reqIter.getAndGoToNext(requestBody)) {
       size_t tmp = reqIdx;
       reqIdx++;
       if (!requestSet.get(tmp)) {
         continue;
       }
-
       ClientRequestMsg req((ClientRequestMsgHeader *)requestBody);
       if (req.flags() & EMPTY_CLIENT_FLAG) {
         continue;
       }
       SCOPED_MDC_CID(req.getCid());
       NodeIdType clientId = req.clientProxyId();
-
       uint32_t actualReplyLength = 0;
       uint32_t actualReplicaSpecificInfoLength = 0;
-      int status;
+      int executionStatus = 1;
```

Comment:
this actually won't work because you'll have the same execution status placeholder for all requests...

Reply:
I'm not passing it by reference, each request has its own execution status.

Reply:
OK, thanks - maybe I misread the code.